test/unit_tests/28_multidepth_objectFifos/single_depth/test.cpp

```diff
@@ -43,8 +43,13 @@ int main(int argc, char *argv[]) {
   int *mem_ptr_in_0 = mlir_aie_mem_alloc(buf0, 32);
   int *mem_ptr_in_1 = mlir_aie_mem_alloc(buf1, 32);
 
+#ifdef AIR_PCIE
+  mlir_aie_external_set_addr_ext_buffer_in_0(buf0.physicalAddr);
+  mlir_aie_external_set_addr_ext_buffer_in_1(buf1.physicalAddr);
+#else
```

Comment:
I think it's clear that we should really fix this interface, rather than proliferate this through yet more tests.

Reply:
Outdated.